test/onnx/test_fx_op_consistency.py

```diff
@@ -737,6 +733,11 @@ def skip_torchlib_forward_compatibility(
         "narrow",
         reason=onnx_test_common.reason_dynamo_does_not_support("data-dependent"),
     ),
+    xfail(
+        "native_batch_norm",
+        dtypes=(torch.float16,),
+        reason="fixme: ORT optimizer error: https://github.com/microsoft/onnxruntime/issues/16438",
```

Comment:
The linked issue is already fixed. Probably need to update 'reason' to reflect latest error.

Reply:
it still fails with float16: onnxruntime.capi.onnxruntime_pybind11_state.Fail: [ONNXRuntimeError] : 1 : FAIL : Type Error: Type (tensor(float16)) of output arg (_native_batch_norm_legit_functional_1) of node (_aten__native_batch_norm_training_functional_onnx_1) does not match expected type (tensor(float)).

Reply:
updated the reason.

Reply:
@BowenBao [scalar_type_analysis](https://github.com/pytorch/pytorch/blob/01abb5af2197cb517fc9b6f241d7dfcc4c66f597/torch/csrc/jit/passes/onnx/scalar_type_analysis.cpp#L49) strikes again?

Reply:
this is fx export. This is a nasty pytorch legacy problem, the torch cpu/gpu behavior doesn't match.

Reply:
https://github.com/microsoft/onnxscript/issues/1269